examples/resources/env0_template/resource.tf

```diff
@@ -7,9 +7,14 @@ data "env0_ssh_key" "my_key" {
 }
 
 resource "env0_template" "example" {
-  name        = "example"
+  name = "example"
```

Comment:
please keep the good formatting, you can use `terraform fmt` for it

Reply:
265ba8f6

Reply:
that commit is exactly the opposite of what I want :D

Reply:
`terraform fmt` will allign all properties to start on the same column

Reply:
I thought GoLang with terraform plugin does that
56d1379a